### PR TITLE
KIN-306 delete product alert products page

### PR DESCRIPTION
**Pull request by FernandoJoachin**
[KIN-306](https://loopcrack.atlassian.net/browse/KIN-306) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

As a company I would like that when its going to be deleted a product, an alert is displayed as a warning so that I can be sure to delete the product and avoid confusions or mistakes on missclick. 

### Solution

Create a modal window to delete the record

### Requirements checklist

- [x] It must show a modal window with a warning message saying that once the product is deleted, there is no way to recover it, and that deleting a product involves deleting its related files and product tags as well.
- [x] It must have a cancel button
- [x] It must have a confirm button
- [x] It must show a response window after confirming the action, this response must indicate if the product was deleted successfully or not.

### Testing

- [x] Delete a product

[KIN-306]: https://loopcrack.atlassian.net/browse/KIN-306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CarlosJCD — CHANGES_REQUESTED** [review]
For some reason the products records does not display on the products view, making us unable to test the delete product alert. Please check if it is something with your changes or we are missing something.

CC: @CarlosMay7

**Comment by CheloChidoone** [comment]
> For some reason the products records does not display on the products view, making us unable to test the delete product alert. Please check if it is something with your changes or we are missing something.
> 
> CC: @CarlosMay7

This doesn't happen to me @CarlosJCD @CarlosMay7, you should try running `php spark migrate:refresh` and `php spark db:seed MainTestSeeder` again

File: app/Views/admin/products/Products.php [hunk 1]
```diff
@@ -98,6 +98,7 @@
                     <!-- card-body -->
                 </div>
                 <!-- card -->
+                <?= view('templates/admin/deleteModalElement', ['idModal' => 'deleteProductsModal', 'message' => 'Eliminar el producto resultará en la eliminación permanente del elemento. Esta acción no se puede deshacer y conlleva la eliminación de los archivos relacionados y los tags asociadas al producto.', 'action' => '/admin/productos/borrar', 'inputName' => 'productId']); ?>
```
Comment (CheloChidoone) [inline app/Views/admin/products/Products.php:101]: I think the message looks better like this:

```suggestion
                <?= view('templates/admin/deleteModalElement', ['idModal' => 'deleteProductsModal', 'message' => 'Eliminar el producto resultará en la eliminación permanente del elemento. Esta acción no se puede deshacer e implica la eliminación de los archivos y los tags asociados al producto.', 'action' => '/admin/productos/borrar', 'inputName' => 'productId']); ?>
```

**Review by CheloChidoone — APPROVED** [review]
Good Job @FernandoJoachin, just check my comment about how you could modify the modal message

**Review by Cristianpan — APPROVED** [review]

**Review by Mauriciocr207 — APPROVED** [review]

**Review by Oswaldo-Chan — COMMENTED** [review]
image

I think the text should be shorter because the box looks more longer.

**Review by CarlosMay7 — APPROVED** [review]

**Review by CarlosJCD — APPROVED** [review]
Nice, it works now. Just check @Oswaldo-Chan comments

**Review by LUGAMAFE — APPROVED** [review]

**Review by LUGAMAFE — APPROVED** [review]

**Pull request merged**